lib/harness/pkg.ex

```diff
@@ -20,9 +20,14 @@ defmodule Harness.Pkg do
   @callback cast(opts :: Keyword.t()) :: t()
 
   @doc """
-  A list of symlinks to create from the .harness directory to project root.
+  A list of links to create from the .harness directory to project root.
```

Comment:
need a line after the period here

like commit messages the first line is the shortdoc